lib/main_common.pm

```diff
@@ -1537,6 +1537,7 @@ sub load_extra_tests_qemu {
 
 sub load_extra_tests_console {
     # JeOS kernel is missing 'openvswitch' kernel module
+    loadtest "console/dracut";
```

Comment:
Don't include dracut test into consoletests, it will break other tests. See https://progress.opensuse.org/issues/47123.

Reply:
sure, I think I won't add this back to consoletest, however it works fine for x86_64 in general. but not for svirt_xen because of issue about serial-terminal.